sqlfmt/common.go

```diff
@@ -0,0 +1,96 @@
+package sqlfmt
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/LUSHDigital/modelgen/model"
+)
+
+// MapType figures out which go type should be used, based on the SQL type.
+func MapType(columnType string, nullable string, typeMap map[string]model.SQLType) (string, error) {
```

Comment:
This is MySQL specific, so will want to go into a MySQL implementation.  (You'll need both).